test/Templates.Test/MvcTemplateTest.cs

```diff
@@ -13,18 +13,14 @@ public MvcTemplateTest(ITestOutputHelper output) : base(output)
         {
         }
 
-        [ConditionalTheory]
+        [ConditionalFact]
         [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
-        [InlineData(null)]
-        [InlineData("F#")]
```

Comment:
It looks like an unrelated change has ended up in this PR. Can we revert the F# change?

Reply:
It impacts the ability for the tests to pass. See the email thread we have about this for details, I can do it in a separate PR if you want me to